SmartDeviceLink/SDLVideoStreamingCodec.h

```diff
@@ -0,0 +1,36 @@
+//
+//  SDLVideoStreamingCodec.h
+//  SmartDeviceLink-iOS
+//
+
+#import "SDLEnum.h"
+
+/**
+ * Enum for each type of video streaming codec
+ */
+typedef SDLEnum SDLVideoStreamingCodec SDL_SWIFT_ENUM;
+
+/**
+ * @abstract SDLVideoStreamingCodec : H264
+ */
+extern SDLVideoStreamingCodec const SDLVideoStreamingCodecH264;
+
+/**
+ * @abstract SDLVideoStreamingCodec : H265
+ */
+extern SDLVideoStreamingCodec const SDLVideoStreamingCodecH265;
+
+/**
+ * @abstract SDLVideoStreamingCodec : THEORA
+ */
+extern SDLVideoStreamingCodec const SDLVideoStreamingCodecTHEORA;
```

Comment:
Based on how other enums are written, it looks like this should be `SDLVideoStreamingCodecTheora`